pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -50,6 +53,7 @@ func NewObjectWatcher(kubeClientSet client.Client, restMapper meta.RESTMapper, c
 		VersionRecord:        make(map[string]map[string]string),
 		RESTMapper:           restMapper,
 		ClusterClientSetFunc: clusterClientSetFunc,
+		defaultExplorer:      defaultexplorer.NewDefaultExplorer(),
```

Comment:
we also need the customized explorer.

Reply:
We shouldn't initialize the explorer here. We can use a shared instance.

Reply:
you mean replace `defaultExplorer` field with below structure that using `NewCustomResourceExplorer` to create 
```
type customResourceExplorerImpl struct {
	kubeconfig string
	informer   informermanager.SingleClusterInformerManager

	customizedExplorer *customizedexplorer.CustomizedExplorer
	defaultExplorer    *defaultexplorer.DefaultExplorer
}
```